ml_metadata/metadata_store/query_config_executor.cc

```diff
@@ -283,11 +283,11 @@ absl::Status QueryConfigExecutor::DowngradeMetadataSource(
 }
 
 std::string QueryConfigExecutor::Bind(const char* value) {
-  return metadata_source_->EscapeString(value);
+  return absl::StrCat("'", metadata_source_->EscapeString(value), "'");
 }
 
 std::string QueryConfigExecutor::Bind(absl::string_view value) {
-  return metadata_source_->EscapeString(value);
+  return absl::StrCat("'", metadata_source_->EscapeString(value), "'");
 }
```

Comment:
This part shouldn't be here, as the new strcat on the metadata source already accounts for escaping.

Reply:
+1 I missed to remove it